estargz.md

```diff
@@ -0,0 +1,349 @@
+# eStargz: Standard-Compatible Extension to Container Image Layers for Lazy Pulling
+
+This doc describes the extension to gzip layers of container images `application/vnd.oci.image.layer.v1.tar+gzip` for *lazy pulling*.
+The extension is called *eStargz*.
+
+eStargz is a *backward-compatible extension* which means that images can be pushed to the extension-agnostic registry and can run on extension-agnostic runtimes.
+
+This extension is based on stargz (stands for *seekable tar.gz*) proposed by [Google CRFS](https://github.com/google/crfs) project (initially [discussed in Go community](https://github.com/golang/go/issues/30829)).
+eStargz extends stargz for chunk-level verification and runtime performance optimization.
```

Comment:
and RFC1952 compliance (https://tools.ietf.org/html/rfc1952#section-2.3.1.1)

Reply:
Fixed.